Index: SlaughterHouse/SlaughterHouse/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/SlaughterHouse/SlaughterHouse/obj/rider.project.model.nuget.info b/SlaughterHouse/SlaughterHouse/obj/rider.project.model.nuget.info
new file mode 100644
--- /dev/null	(date 1759476975219)
+++ b/SlaughterHouse/SlaughterHouse/obj/rider.project.model.nuget.info	(date 1759476975219)
@@ -0,0 +1,1 @@
+17594769611488184
\ No newline at end of file
